components/item/MediaThumbnailGrid.vue

```diff
@@ -4,18 +4,23 @@
     class="d-flex flex-wrap mb-3"
   >
     <!-- TODO: populate alt, but with what? -->
-    <b-img-lazy
+    <a
       v-for="(thumbnail, index) of thumbnails"
       :key="index"
-      :src="thumbnail.src"
       :class="thumbnailImgClass(thumbnail, index)"
       :data-about="thumbnail.about"
-      :data-qa="`media thumbnail #${index + 1}`"
-      thumbnail
-      alt=""
-      class="mb-2 mr-2 rounded-0"
-      @click.native="clickThumbnail(thumbnail.about)"
-    />
+      :data-qa="`link thumbnail #${index + 1}`"
+      href="#"
```

Comment:
href is not a required attribute. As this anchor element does not link anywhere, remove the href attribute.